spec/features/create_virtual_object_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Use Argo to create a virtual object with constituent objects', type: :feature do
+  let(:start_url) { Settings.argo_url }
+  let(:filename_group) { ['example.tiff', 'example.jp2'] }
+  let(:csv_path) { File.join(DownloadHelpers::PATH, 'virtual-object.csv') }
+  let(:virtual_objects_description) { RandomWord.phrases.next }
+  let(:num_constituents) { 11 }
```

Comment:
Does the number have to be so high to test this feature?  Would 2 constituents be enough?  It's valuable to have tests, but having tests that run fast is important too or people will not want to use this tool.

Reply:
This number was requested by Andrew, but @lwrubel was going to perform some time testing to determine if it is realistic.

Reply:
14 minutes is how much time it takes to run the rest of the suite, so I'd much prefer we use 2 if that's okay.

Reply:
I think it would be useful to have an automated test that takes longer than the usual integration test but doesn't have to be run all the time. I would not test dedicated changes to virtual objects by using only two constituents, so if this test only uses two it would not take the place of testing that I do manually.

Reply:
Thanks for that context, @andrewjbtw. Would the approach of running it in a way that's configurable, such as `SETTINGS__NUMBER_OF_CONSTITUENTS=11 bundle exec rspec` like @jcoyne suggested work? @mjgiarlo you were also interested in this test, I think, so I'm wondering if this would also work for what you were envisioning.

Reply:
Yes - configurable would be great.

Reply:
@lwrubel hi! any number of constituents would suffice for me.